app.cfg.example

```diff
@@ -100,6 +100,9 @@ upload_policy = once
 # value can be a space delimited list of GH accounts
 deploy_permission = trz42
 
+[buildcfg]
+# which GH account has the permission to trigger the build 
+build_permission = Hafsa-Naeem
```

Comment:
Let's add `build_permission` to the existing `buildenv` section, rather than introducing a new `buildcfg` section